draft-ietf-sedate-datetime-extended.md

```diff
@@ -190,134 +190,60 @@ timestamp extension suffix and defines such a format that extends
 The format should allow implementations to specify additional
 important information in addition to the bare timestamp.
 This is done by defining *tags*, each with a *key* and
-a *value* separated by an equals sign, and
-allowing implementations to include an informative
+a *value* separated by an equals sign.
+The key of a tag can be split by including a hyphen/minus sign "`-`".
+The value of a tag can be a hyphen/minus delimited list of multiple values.
+
+Out of these tags, applications can build an informative
 *suffix* at the end with as many tags as required.
-The value of a tag can be a hyphen delimited list of multiple values.
 
-In case a key is repeated or conflicted, implementations MUST give
-precedence to whichever value is positioned first.
-<!-- needs example and/or defn of "conflicted" -->
+Keys are case-sensitive.  Values are case-sensitive unless otherwise specified.
+
+In case a suffix repeats a key or otherwise contains conflicting tags,
+implementations MUST give precedence to whichever value is positioned
+first. [^interop1]
+
+[^interop1]:  I'd rather place a MU⁠ST NOT for this case, first.
```

Comment:
the problem with any MUST NOT when generating values is that you need to define how to parse incorrect inputs, and the choices are either "reject outright", "undefined" or something like what we have here.

Reply:
Yes, but defining it as a MUST NOT is a prerequisite to fewer people generating it.

(The funny space in MUST NOT is working around an RFCXMLv3 grammar bug.)

Reply:
Yes, we need more text about error handling.
What is there now translates into "put in conflicting information at will, the recipient will sort it out", which usually leads into an interop nightmare, because they won't (at least not interoperably).
Added a little todo to that text now.

Reply:
I've seen this before as "creators MUST NOT generate" and "if seen, recipients MUST disambuguate as follows"

(I meant to type disambiguate, but I do like disambuguate as a word)

Reply:
I'll put disambuguate into the definitions :-)
Yes, the combo would be the right thing.
The second MUST still heavily encourages ignoring the first, though.